src/adhocracy_core/docs/authentication_api.rst

```diff
@@ -385,3 +385,67 @@ the user to be logged in from different devices at the same time. ::
     >>> resp_data = testapp.get('/meta_api/', headers=headers).json
     >>> 'resources' in resp_data.keys()
     True
+
+Alternative A: User Password reset without API extension (more backend friendly)
+--------------------------------------------------------------------------------
+
+A password reset link is send by creating a password reset object under the
+``/principals/passwordresets`` pool. The user is identified with the given
+email adress::
+
+    >>> prop = {'content_type': 'adhocracy_core.resources.principal.IPaswordReset',
+    ...         'data': {
+    ...              'adhocracy_core.sheets.principal.IPasswordReset': {
+    ...                  'email': 'anna@example.org'}}}
+    >>> resp_data = testapp.post_json(rest_url + "/principals/passwordresets", prop).json
+
+the url for the created password reset object is protected by obscurity::
```

Comment:
This confuses me as well - if the password reset resource ID *is* the token, it shouldn't be returned in the response - otherwise I could reset anybody's password...

Did we understand something wrong?